docs/en/latest/how-to-build.md

```diff
@@ -119,7 +119,11 @@ Follow the steps below to install Apache APISIX via the source release package.
 
   **Note**: If `make deps` fails with "install `lualdap` failed" with an error like `Could not find header file for LDAP` try the solution below.
 
-  Solution: Set `LDAP_DIR` with `luarocks config` manually. For example `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`.
+      Solution: luarocks supports adding custom package addresses to the project. You can set `LDAP_DIR` with `luarocks config` manually, for example `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`. You should config as follows:
+      1. Install `openldap` locally using tools including but not limited to 'brew';
+      2. Locate the 'openldap' local installation directory. Note that the default address for 'brew' in macOS for ARM architecture is '/opt/homebrew/cellar/', and macOS for x86 architecture defaults to '/usr/local/opt/';
```

Comment:
Can we give a command to locate the directory? Or we may have to update this statement after the new macos release.

Reply:
Google searched, and there is indeed a command which you can use to find the installation package.
Thanks! modify the documentation now!